### ISSUE TITLE
CPU time limit reached

### ISSUE THREAD

**Issue by saptarshiguha** [randy3k/radian#67]
Hello,
While running rtichike on an EC2 instance (With no cpu limits), i sometimes get this error

```
Error in if (.isMethodsDispatchOn() && methods::is(genfun, "genericFunction")) genfun <- methods::selectMethod(genfun,  :
  reached CPU time limit
```

nothing happens and i can continue running my code, but this seems odds.

**Comment by randy3k** [comment]
It is related to the cancellation of the completions if it is too slow. The message should be normally suppressed. Not sure why it shows up in your case.

**Comment by randy3k** [comment]
I finally figured out the reason why this error message was not suppressed. It should be fixed in v0.4.1.
